antithesis_sdk.h

```diff
@@ -30,8 +32,9 @@ namespace antithesis {
     };
 
     struct JSON;
-
-    typedef std::variant<std::string, bool, char, int, uint64_t, float, double, const char*, JSON> ValueType;
+    typedef std::variant<std::string, bool, char, int, uint64_t, float, double, const char*, JSON> BasicValueType;
+    typedef std::vector<antithesis::BasicValueType> JSON_ARRAY;
```

Comment:
Very minor nit: You're already in the `antithesis` namespace, so you don't need `antithesis` here.

Reply:
will fix - thank you.